docs/roadmap/2023-11-dbt-tng.md

```diff
@@ -0,0 +1,107 @@
+# dbt: TNG (November 2023)
```

Comment:
If we go with "TNG" as the title, can we write it out fully as "the next generation"

Reply:
good call!